index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <script src="./index.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@500&display=swap" rel="stylesheet">
+    <title>Weather App</title>
+</head>
+<body>
+    <div class="card">
+    <div class="search">
+    <input type="text" class="search-bar" placeholder="search">
+   <button type="submit" style="height: 25px; width: 60px;">Search</button>
+    </div>
+
+    <div class="weather loading">
+    <h2 class="sity">Weather in Belgrade</h2>
```

Comment:
Please fix spelling of  "sity" and there is no need to prefill these tags with strings (Weather in Belgrade, 39c ,etc...)